WORKSPACE

```diff
@@ -100,14 +100,14 @@ load("//defs:repo_rules.bzl", "fetch_kube_release")
 fetch_kube_release(
     name = "io_k8s_release",
     archives = {
-        "kubernetes-server-linux-amd64.tar.gz": "d8c7d4e98355be59b4ecece8ef15a55ef48d180f9ee0ac86f81e9f4166232657",
+        "kubernetes-server-linux-amd64.tar.gz": "af62692a5d5c6024a78d230061061b3fcd2f5422cea2f3d6aaad2fe20fb13746",
         #"kubernetes-server-linux-amd64.tar.gz": "1772c20e7765a2288bd335f754b9cf5cb2e2e0b5b11f80db0d9b582c1016c663",
```

Comment:
Is there a reason we're hanging on to these legacy versions in a comment? Seems like it should be safe to clean these up.

Reply:
Updated